tests/Aspire.Dashboard.Components.Tests/Pages/StructuredLogsTests.cs

```diff
@@ -124,6 +125,7 @@ public void Render_FiltersWithSpecialCharacters_SuccessfullyParsed()
             builder.Add(p => p.ViewportInformation, viewport);
         });
 
+        await Task.Delay(2000);
```

Comment:
Hmmm. You shouldn't do this unless it's nessessary. Is it nessessary?
If you do do it, there should be a comment saying why.